test/order.js

```diff
@@ -68,5 +68,41 @@ describe('Fast-Food-Fast', () => {
           done();
         });
     });
+    it('it should update order with ID 3', (done) => {
```

Comment:
Use a generic title for your test-case descriptions. `it should successfully update order given an Id` seems like a superior title to `it should update order with ID 3`.

Reply:
Ok this commit has been fixed